cmake/create_test.cmake

```diff
@@ -219,6 +221,12 @@ function(ginkgo_add_test test_name test_target_name)
             PROPERTIES ENVIRONMENT LD_PRELOAD=${test_preload}
         )
     endif()
+    if(add_test_LABEL)
+        set_tests_properties(
+            ${REL_BINARY_DIR}/${test_binary_name}
+            PROPERTIES LABELS "${add_test_LABEL}"
+        )
+    endif()
```

Comment:
nice find - seems supported since CMake 3.16, so I think this could be useful. We could also use it to add executor information